### PR TITLE
Location naming changes

### PR DESCRIPTION
**Pull request by sheelio**
@esoergel @amsagoff 

Ethan, I don't know how to change the text "Location Type" on the edit or create location pages (i.e. https://www.commcarehq.org/a/location-documentation/settings/locations/314dcabfc83885a7bcf609177506966f/).  

Other than that, this is good to go.

### CODE REVIEW

File: corehq/apps/locations/forms.py [hunk 1]
```diff
@@ -48,7 +48,11 @@ class LocationForm(forms.Form):
         widget=ParentLocWidget(),
     )
     name = forms.CharField(max_length=100)
-    location_type = forms.CharField(widget=LocTypeWidget(), required=False)
+    location_type = forms.CharField(
+        label=_('Organization Level'),
+        required=False,
+        widget=LocTypeWidget(),
```
Comment (dimagimon) [inline corehq/apps/locations/forms.py]: W291 trailing whitespace

File: corehq/apps/locations/forms.py [hunk 2]
```diff
@@ -47,9 +47,17 @@ class LocationForm(forms.Form):
...
-    name = forms.CharField(max_length=100)
-    location_type = forms.CharField(widget=LocTypeWidget(), required=False)
+    name = forms.CharField(
+        label=_('Name'),
+        max_length=100,
+    )
+    location_type = forms.CharField(
+        label=_('Organization Level'),
+        required=False,
+        widget=LocTypeWidget(),
```
Comment (dimagimon) [inline corehq/apps/locations/forms.py]: W291 trailing whitespace

**Comment by czue** [comment]
@sheelio look at you and your bad self!

**Comment by czue** [comment]
um, i assumed this was good to go from a UI changing side. @dimagi/product let me know if that wasn't the case and we'll revert.

**Comment by czue** [comment]
(and @sheelio forgot to mention but you should use the "open for review, don't merge" label if you don't want that to happen in the future)

**Comment by sheelio** [comment]
Cool thanks - this good to merge.

**Comment by esoergel** [comment]
Looks solid!

**Pull request merged**